simulator/src/main/java/com/hazelcast/simulator/coordinator/Coordinator.java

```diff
@@ -387,21 +411,76 @@ private void runSequential() {
         }
     }
 
-    private void moveLogFiles() {
-        if (isLocal(simulatorProperties)) {
-            File workerHome = newFile(getSimulatorHome(), WORKERS_HOME_NAME);
-            File targetDirectory = ensureExistingDirectory(newFile(".", WORKERS_HOME_NAME), testSuite.getId());
-
-            String workerPath = workerHome.getAbsolutePath();
-            String targetPath = targetDirectory.getAbsolutePath();
-
-            execute(format("mv %s/%s/* %s || true", workerPath, testSuite.getId(), targetPath));
-            execute(format("rmdir %s/%s || true", workerPath, testSuite.getId()));
-            execute(format("rmdir %s || true", workerPath));
-            execute(format("mv ./agent.err %s/ || true", targetPath));
-            execute(format("mv ./agent.out %s/ || true", targetPath));
-            execute(format("mv ./failures-%s.txt %s/ || true", testSuite.getId(), targetPath));
+    private void downloadLocal() {
+        File workerHome = newFile(getSimulatorHome(), WORKERS_HOME_NAME);
+        File targetDirectory = ensureExistingDirectory(newFile(".", WORKERS_HOME_NAME), testSuite.getId());
+
+        String workerPath = workerHome.getAbsolutePath();
+        String targetPath = targetDirectory.getAbsolutePath();
+
+        execute(format("mv %s/%s/* %s || true", workerPath, testSuite.getId(), targetPath));
+        execute(format("rmdir %s/%s || true", workerPath, testSuite.getId()));
+        execute(format("rmdir %s || true", workerPath));
+        execute(format("mv ./agent.err %s/ || true", targetPath));
+        execute(format("mv ./agent.out %s/ || true", targetPath));
+        execute(format("mv ./failures-%s.txt %s/ || true", testSuite.getId(), targetPath));
```

Comment:
The hdr files are already in the right place because the hdrcontainer gets the output directory as argument.